specific-book/main.js

```diff
@@ -1,23 +1,21 @@
 "use strict";
-const imgWrapper = document.querySelector(".svg-wrapper");
-const prise = document.querySelector(".prise");
-const totalPrise = document.querySelector(".total-prise");
-const input = document.querySelector("#input-count");
-const inputs = document.querySelectorAll("#input-count");
-const imgUp = document.querySelector(".img-svg-up");
-const imgDown = document.querySelector(".img-svg-down");
-const button = document.querySelector(".my-button-style");
+const imgButtonsForForm = document.querySelector(".svg-wrapper");
+const formPriseValue = document.querySelector(".prise");
+const formTotalPrise = document.querySelector(".total-prise");
+const inputFormOrdering = document.querySelector("#input-count");
+const inputsFor42MaxLogic = document.querySelectorAll("#input-count");
```

Comment:
id must be unique, I think you need to find it by className. And also don't use quantity in variable names. I think you don't need this input at all. - you have only one input on the page

Reply:
<img width="1124" alt="image" src="https://user-images.githubusercontent.com/2098199/224134989-be5fe61e-bc67-4e9f-b426-a132c2464178.png">